ansible/ci-playbooks/roles/ci-base-setup/tasks/redhat.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: ci-base-setup (redhat) | Set hostname
+  hostname:
+    name: "{{ inventory_hostname }}"
```

Comment:
The hostname is automatically set by cloud-init when you spin up a new instance, there shouldn't be an "initial" name.